app_test.go

```diff
@@ -477,3 +478,260 @@ func TestGetAppNoTarget(t *testing.T) {
 		t.Fatal("unexpected <nil> error")
 	}
 }
+
+func TestSetConfig(t *testing.T) {
+	fakeServer := newFakeServer(t)
+	defer fakeServer.stop()
+	fakeServer.prepareResponse(preparedResponse{
+		code:    http.StatusOK,
+		method:  "POST",
+		path:    "/apps/proj1-prod/env",
+		payload: []byte("{}"),
+	})
```

Comment:
@fsouza it just occurred to me it may be interesting to have some integration tests in Tranor as well. We've been mocking all the calls to Tsuru API and I do have the concern Tsuru may change its interface and all tests of Tranor would keep working - but the endpoints would potentially be broken.
As Tranor is open source, this could be an optional suite which could get from an environment variable the Tsuru instance we'd like to use to run them, for instance. What are your thoughts on this? @aflag & @Lepthy, please, also share your thoughts in this regard.